docs/operating-system/debugging.md

```diff
@@ -9,21 +9,27 @@ This section is not for end users. End users should use the [SSH add-on] to SSH
 
 [SSH add-on]: https://github.com/home-assistant/hassio-addons/tree/master/ssh
 
-## SSH access to the host
+## SSH access
+
+Home Assistant has two different types of ssh access.
+- End user SSH on port 22. This port is used by the [SSH add-on], it provides easy but limited SSH access to the system.
```

Comment:
This is actually disabled by default and not required or related to debugging host access. I suggest to remove it.

Reply:
It is required when using the CLI to load the authorized_keys file via `ha os import`